tests/test_javascript_server.py

```diff
@@ -140,3 +149,51 @@ def test_run_logging(tmpdir):
     assert (
         validate_ping.validate_ping(input, output, schema_url=schema_url) == 0
     ), output.getvalue()
+
+
+@pytest.mark.node_dependency
+def test_logging_events_ping_with_event_metrics(tmpdir):
+    tmpdir = "/tmp/glean-js-server"
+    tmpdir = Path(str(tmpdir))
+
+    translate.translate(
+        [
+            ROOT / "data" / "ruby_server_metrics.yaml",
```

Comment:
I think it would also make sense to rename `ruby_server_metrics.yaml` to something like `server_metrics_with_event.yaml`. This definition isn't Ruby-specific and after this PR will be used in js test.

Reply:
It would

Reply:
Let me do that before merging.